web/src/components/views/Transform.vue

```diff
@@ -115,6 +114,5 @@ v-container(fill-height)
                 :value="m.value", :key="`scale${m.value}`")
     v-layout.pa-2(column)
       v-card
-        img(:src="`${boxUrl}?cachebust=${norm}${trans}${scaling}`", style="width: 100%;")
```

Comment:
Before we actually get rid of this, I'd like to have a quick discussion about the value of the plot itself. If it's needed in the app, we should talk about requirements for a better version; if it was simply a vis placeholder, then let's get rid of it. (A comment made by maybe Jeff made me think we *do* need a boxplot on this screen.)

Reply:
@jeffbaumes do we need a boxplot on this screen?   My potentially faulty memory is that I pulled the boxplot out of out of metaboanalyst, but that we were more focused on a PCA.

That box plot had badly labeled axis and looked terrible.  If we actually need one, I agree with @waxlamp  that we should open a new issue for one to do in D3.

Reply:
Yeah, I'd say to get rid of the box plot. If it is requested, we should find a way to make a dynamic box plot client-side.

Reply:
Thanks @jeffbaumes!